apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V100__ccw_eoy_2022_audit_changes.sql

```diff
@@ -0,0 +1,81 @@
+-- This flyway script is intended to track changes to the beneficiary_monthly
+-- table, primarily during the first couple of weeks of a new calendar year.
+-- This enables BFD to track data changes to the beneficiary_monthly table;
+-- visibility into BENEFICIARY ref_yr changes do not effect previous year's data.
+
+${logic.psql-only} DROP TABLE IF EXISTS public.beneficiary_monthly_audit;
+${logic.psql-only} DROP TRIGGER IF EXISTS audit_ccw_update ON beneficiary_monthly;
+
+-- Create beneficiary_monthly_audit table; we'll create the table in both HSQL & PSQL
+-- but we don't bother with setting up the trigger in HSQL as this will only be useful
+-- during that timeframe when we are actively vetting CCW end-of-year/new year data.
+--
+CREATE TABLE public.beneficiary_monthly_audit(
+  bene_id                                bigint NOT NULL,
+  year_month                             date NOT NULL,
+  partd_contract_number_id               varchar(5),
+  partc_contract_number_id               varchar(5),
+  medicare_status_code                   varchar(2),
+  fips_state_cnty_code                   varchar(5),
+  entitlement_buy_in_ind                 char(1),
+  hmo_indicator_ind                      char(1),
+  medicaid_dual_eligibility_code         varchar(2),
+  partd_pbp_number_id                    varchar(3),
+  partd_retiree_drug_subsidy_ind         char(1),
+  partd_segment_number_id                varchar(3),
+  partd_low_income_cost_share_group_code varchar(2),
+  partc_pbp_number_id                    varchar(3),
+  partc_plan_type_code                   varchar(3),
+  operation                              char(1) NOT NULL,
+  update_ts                              timestamp NOT NULL,
+ CONSTRAINT beneficiary_monthly_audit_pkey PRIMARY KEY (bene_id, year_month));
```

Comment:
When I tested locally and modified the same `beneficiary_monthly` record twice the update failed because of a primary key constraint violation on `beneficiary_monthly_audit_pkey`.  Is that expected?